lib/OpenQA/Utils.pm

```diff
@@ -875,6 +875,10 @@ sub read_test_modules {
     my $testresultdir = $job->result_dir();
     return unless $testresultdir;
 
+    my $modules_spent_time_file = path($testresultdir, "modules_spent_time.json");
```

Comment:
I would prefer to read the data from the existing test module results file rather than a new one.